examples/get_latest_sensor_value.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# Copyright 2018 SKA South Africa (http://ska.ac.za/)
+# BSD license - see COPYING for details
+"""Simple example demonstrating retrieving the latest sensor value.
+
+This example gets lists of sensor names, then returns the
+latest value of each specific sensor.  It uses HTTP access to katportal.
+"""
+import logging
+import argparse
+
+import tornado.gen
+
+from katportalclient import KATPortalClient
+from katportalclient.client import SensorNotFoundError
+
+
+logger = logging.getLogger('katportalclient.example')
+logger.setLevel(logging.INFO)
+
+
+@tornado.gen.coroutine
+def main():
+    # Change URL to point to a valid portal node.
+    portal_client = KATPortalClient('http://{}/api/client'.format(args.host),
+                                    on_update_callback=None, logger=logger)
+
+    # Get the names of sensors matching the patterns
+    # See examples/get_sensor_info.py for details on sensor name pattern matching
+    sensor_names = yield portal_client.sensor_names(args.sensors)
+    print "\nMatching sensor names for pattern {}: {}".format(args.sensors, sensor_names)
+
+    # Fetch the readings for the sensors found.
+    if len(sensor_names) == 0:
+        print "No matching sensors found!"
+    else:
+        for sensor_name in sensor_names:
+            try:
+                sensor_value = yield portal_client.sensor_value(sensor_name,
+                                                                include_value_ts=True)
+            except SensorNotFoundError as exc:
+                print "\n", exc
```

Comment:
Not sure what the "exc" looks like by it may have been good to include the sensor_name in this print (although it probably is in the exception). No need to change just this, though.

Reply:
Yes, the sensor name is in the string, see https://github.com/ska-sa/katportalclient/pull/34/files#diff-6355b287a79bb44c686cf1256475bf01R1256